src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -110,6 +110,7 @@ public function onKernelResponse(FilterResponseEvent $event)
      * Injects the web debug toolbar into the given Response.
      *
      * @param Response $response A Response instance
+     * @param Request  $request  A Request instance
```

Comment:
Instead, I would remove the previous @param as these phpdocs do not add any values.